src/main/request.ts

```diff
@@ -135,6 +135,24 @@ export class Request {
         });
     }
 
+    async sendBlob(method: string, url: string, contentType: string, options: RequestOptions = {}) {
+        const { body, query, headers } = options;
+        const res = await this.send(method, url, {
+            headers: {
+                'content-type': contentType,
+                ...headers,
+            },
+            query,
+            body: body ? JSON.stringify(body) : null,
+        });
+        const { status } = res;
+        if (status === 204 || res.headers.get('content-length') === '0') {
+            return null;
+        }
+
+        return await res.blob();
```

Comment:
this probably makes sense only for get blob.

Reply:
Yeah, it is what it is. GET only then

![dont-know-shrug-gif-by-copenhagen-flames](https://github.com/ubio/request/assets/64712584/8fff1e33-13cc-43e1-8efc-beffadb87c66)